device-types/Sophos/XGS-2100.yaml

```diff
@@ -0,0 +1,51 @@
+---
+manufacturer: Sophos
+model: XGS 2100
+slug: sophos-xgs-2100
+u_height: 1
+is_full_depth: false
+subdevice_role: parent
```

Comment:
`subdevice_role` is used for parent child relationships for devices that slot into one another. For instance, a Dell MX620 (child) slots into a Dell VRTx (parent). Thus, there are `device-bays`. I do not see any `device-bays` on this device. Please remove this definition if this is not in fact a chassis for another unit

Reply:
Thanks for sanity checking, I seemingly was confused with device-bays vs. module-bays, subdevice_role is nor removed.